translations/formation.fr.yml

```
@@ -1,30 +1,56 @@
 formation:
-  categories: Categories
+  actico: Activité Commerciale
```

```suggestion
  activite_commerciale: Activité Commerciale
```
Désolé, je l'avais loupé.